types/initrc/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for initrc 1.2
+// Project: https://github.com/Soldy/initrc
+// Definitions by: Soldy <https://github.com/Soldy>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+export class Init {
+    constructor();
+    startAdd(fun: () => void, level: number, name: string): boolean;
+    startRun(): Promise<void>;
```

Comment:
```suggestion
    /** @async */
    startRun(): Promise<void>;
```
https://jsdoc.app/tags-async.html